src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Boolean.txt

```diff
@@ -46,14 +43,20 @@ true
 >> Boolean(2)
 true
 
->> Boolean([0, 1, 2])
-Table({Value:false},{Value:true},{Value:true})
-
->> Boolean([])
-Errors: Error 0-11: The function 'Boolean' has some invalid arguments.|Error 8-10: Invalid schema, expected a one-column table.
-
 >> Boolean(true)
-Errors: Error 0-13: The function 'Boolean' has some invalid arguments.|Error 8-12: Invalid argument type (Boolean). Expecting a Text value instead.
+true
 
 >> Boolean(false)
-Errors: Error 0-14: The function 'Boolean' has some invalid arguments.|Error 8-13: Invalid argument type (Boolean). Expecting a Text value instead.
\ No newline at end of file
+false
+
+>> Boolean(true Or true)
```

Comment:
@CarlosFigueiraMSFT  - these .txts cover the question you asked int he PAClient PR.